vrs_anvil/cli.py

```diff
@@ -68,17 +81,90 @@ def cli(ctx, verbose: bool, manifest: str, max_errors: int):
 
 
 @cli.command("annotate")
+@click.option('--scatter', help='Start background process per url', required=False, default=False,
+              is_flag=True, show_default=True)
 @click.pass_context
-def annotate_cli(ctx):
+def annotate_cli(ctx, scatter: bool):
     """Read manifest file, annotate variants, all parameters controlled by manifest.yaml."""
 
+    timestamp_str = ctx.obj["timestamp_str"]
+
+    if not scatter:
+        try:
+            assert "manifest" in ctx.obj, "Manifest not found."
+            manifest = ctx.obj["manifest"]
+            _logger.debug(f"Manifest: {ctx.obj['manifest']}")
+            click.secho("🚧  annotating variants", fg="yellow")
+            metrics_file = annotate_all(manifest, max_errors=ctx.obj["max_errors"], timestamp_str=timestamp_str)
+            click.secho(f"📊  metrics available in {metrics_file}", fg="green")
+        except Exception as exc:
+            click.secho(f"{exc}", fg="red")
+            _logger.exception(exc)
+    else:
+        try:
+            assert "manifest" in ctx.obj, "Manifest not found."
+            parent_manifest = ctx.obj["manifest"]
+            c = 0
+            scattered_processes = []
+            for _ in parent_manifest.vcf_files:
+                child_manifest = Manifest.parse_obj(parent_manifest.dict())
+                child_manifest.vcf_files = [_]
+                child_manifest.num_threads = 1
+                child_manifest.disable_progress_bars = True
+                child_manifest_path = pathlib.Path(child_manifest.work_directory) / f"manifest_{c}.yaml"
```

Comment:
Is it beneficial to store the child manifests with a timestamp so they don't get overwritten? Without it, I think we may not have a complete/unified set of data to use downstream for analyses if annotate is rerun. For example, if Bob the informatician edits the main `manifest.yaml` for a `vrs_anvil annotate` rerun, that manifest is out of date with old results and there's no way to reference in a file the params he used. Storing the manifest with a timestamp means that he can locate all the files and info associated with that run not just the most recent as vrs_anvil ps does.

Maybe a contrived example, but including timestamps also allows potential to be add functionality to do something like `vrs_anvil ps` for a given past timestamp.

Reply:
done